src/sections/detailBoxes/detailBoxes.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import DetailBox from "./detailBox";
+import styled from "styled-components";
+import tw from "twin.macro";
+
+const BOX_TEXT = [
+    {
+        title: "Connect",
+        topText: "with like-minded students.",
+        bottomText: "with 500+ students from across Canada",
+        backgroundUrl: "/detailBoxes/connect.webp"
+    },
+    {
+        title: "Learn",
+        topText: "from world-renowned experts.",
+        bottomText: "from etc etc",
+        backgroundUrl: "/detailBoxes/learn.webp"
+    },
+    {
+        title: "Discover",
+        topText: "career opportunities",
+        bottomText: "test text",
```

Comment:
Discover opportunities with many of the companies sponsoring us and explore what your career could look like in the future.